packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -31,6 +31,12 @@
 			overflow: visible;
 		}
 	}
+
+	> .components-resizable-box__container {
```

Comment:
Do we need the `>` operator here? I'd prefer not to use this operator unless we explicitly only want to target the direct children.

Reply:
I think it should work without the `>` operator as well. I will push the update.

Reply:
Updated in 3872947.